images/regen-sandbox/README.md

```diff
@@ -0,0 +1,29 @@
+# Regen Ledger Sandbox
+
+This container (and bootstrapping scripts) is intended to aid with end-to-end testing of client libraries, applications, manual local network tests, and other scenarios where you want to be able to quickly bootstrap a single node regen network (targeting any version or commit hash of regen-ledger), and populate the network with some basic data for your e2e testing or exploratory purposes.
+
+
+Building locally:
+```
+cd regen-ledger
+docker build . -f images/regen-sandbox/Dockerfile -t regen-sandbox
+```
+
+Running the container does the following:
+```
+cd regen-ledger/images/regen-sandbox
+export REGEN_MNEMONIC="YOUR TESTING MNEMONIC"
```

Comment:
i'm unsure about what mnemonic should be used here. can you add some lines to describe this some more?

Reply:
@wgwz and I spoke abt this on a call, and I think I'll also add support for generating a mnemonic within the container, and just printing it to STDOUT, so that if a user wants, they can rely on the regen ledger process for generating a mnemonic.

Reply:
Fixed in [ca32a57](https://github.com/regen-network/regen-ledger/pull/1550/commits/ca32a5791160d99fe192d42dca585d7d7ae3a361)